lib/galaxy/datatypes/util/maf_utilities.py

```diff
@@ -85,6 +85,7 @@ def get_open_tempfile(self, index=None, **kwds):
             if index is None:
                 index = len(self.files)
                 temp_kwds = dict(self.kwds)
+                temp_kwds['delete'] = False
```

Comment:
That makes sense, I assume `__del__ ` is supposed to delete the files. If you don't do that then `tmp_file.close()` will delete the file.

Reply:
Yeah, the class overloads `__del__` to pass `delete=True` to the `close()` call